pkg/util/lifted/corevalidation.go

```diff
@@ -36,7 +36,7 @@ import (
 )
 
 // +lifted:source=https://github.com/kubernetes/kubernetes/blob/release-1.22/pkg/apis/core/validation/validation.go#L59
-const isNegativeErrorMsg string = apimachineryvalidation.IsNegativeErrorMsg
+const isNegativeErrorMsg = apimachineryvalidation.IsNegativeErrorMsg
```

Comment:
Please don't touch the `lifted` code if it is not a bug.

Reply:
Same as below.

Reply:
Got it.